pkg/registry/core/serviceaccount/storage/token.go

```diff
@@ -199,15 +232,19 @@ func (r *TokenREST) GroupVersionKind(schema.GroupVersion) schema.GroupVersionKin
 }
 
 // newContext return a copy of ctx in which new RequestInfo is set
-func newContext(ctx context.Context, resource, name string, gvk schema.GroupVersionKind) context.Context {
-	oldInfo, found := genericapirequest.RequestInfoFrom(ctx)
-	if !found {
-		return ctx
+func newContext(ctx context.Context, resource, name string, gvk schema.GroupVersionKind, clusterScoped bool) context.Context {
```

Comment:
actually, I think it'll be clearer to pass in the namespace to use:

```suggestion
func newContext(ctx context.Context, resource, name, namespace string, gvk schema.GroupVersionKind) context.Context {
```

and explicitly set it to "" when looking up nodes and set it to `namespace` (which we already look up and verify at the top of `Create`) when looking up other things

Reply:
then we don't have to call RequestInfoFrom at all in this method